Code/skyrim_ui/src/assets/i18n/de.json

```diff
@@ -139,6 +139,9 @@
         "WRONG_PASSWORD": "Das eingegebene Passwort ist falsch.",
         "NO_REASON": "Der Server hat die Verbindung ohne Angabe von Gründen verweigert."
       }
+    },
+    "PLAYER_LIST": {
+      "PARTY_INVITE": "{{from}} hat Sie zu einer Gruppe eingeladen"
```

Comment:
I would use the translation below as it isn't as formal

`{{from}} hat dich in eine Gruppe eingeladen`

Reply:
changed